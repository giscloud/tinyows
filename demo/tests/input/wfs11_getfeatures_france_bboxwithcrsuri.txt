SERVICE=WFS&REQUEST=GetFeature&VERSION=1.1.0&TYPENAME=tows:world&bbox=222638,6274861,333958,6446275,urn:ogc:def:crs:EPSG::3857
