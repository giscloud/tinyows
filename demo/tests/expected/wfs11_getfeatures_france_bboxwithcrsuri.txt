Content-Type: text/xml; subtype=gml/3.1.1

<?xml version='1.0' encoding='UTF-8'?>
<wfs:FeatureCollection
 xmlns:tows='http://www.tinyows.org/'
 xmlns:wfs='http://www.opengis.net/wfs'
 xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
 xmlns:gml='http://www.opengis.net/gml'
 xmlns:xsd='http://www.w3.org/2001/XMLSchema'
 xmlns:ogc='http://www.opengis.net/ogc'
 xmlns:xlink='http://www.w3.org/1999/xlink'
 xmlns:ows='http://www.opengis.net/ows'
 xsi:schemaLocation='http://www.tinyows.org/
    http://127.0.0.1/cgi-bin/tinyows?service=WFS&amp;version=1.1.0&amp;request=DescribeFeatureType&amp;Typename=tows:world   http://www.opengis.net/wfs
   http://schemas.opengis.net/wfs/1.1.0/wfs.xsd
   http://www.opengis.net/gml
   http://schemas.opengis.net/gml/3.1.1/base/gml.xsd'
>
  <gml:featureMember>
   <tows:world gml:id="world.62">
   <tows:name>France</tows:name>
   <tows:gmi_cntry>FRA</tows:gmi_cntry>
   <tows:region>Europe</tows:region>
   <tows:geom><gml:MultiSurface srsName="urn:ogc:def:crs:EPSG::4326"><gml:surfaceMember><gml:Polygon><gml:exterior><gml:LinearRing><gml:posList>41.366379 9.219166 41.392769 9.0975 41.474709 9.080555 41.506649 8.882221 41.588871 8.779999 41.632488 8.794722 41.67638 8.913887 41.69804 8.878887 41.751659 8.680275 41.776932 8.724722 41.894428 8.802219 41.93166 8.754164 41.89777 8.614721 41.966099 8.589441 41.978039 8.658609 42.040821 8.726387 42.10944 8.696665 42.14888 8.561943 42.214161 8.572777 42.25499 8.641109 42.27026 8.688055 42.349991 8.612221 42.376659 8.555832 42.461929 8.672222 42.507771 8.663332 42.636379 8.937498 42.6586 9.053053 42.710819 9.083887 42.73082 9.210554 42.673321 9.290833 42.73415 9.34111 42.894989 9.320833 42.958881 9.346388 43.00082 9.344721 43.005821 9.430277 42.927212 9.465832 42.81443 9.489443 42.6036 9.459999 42.56636 9.502857 42.600819 9.497221 42.429989 9.543055 42.06554 9.508055 41.906651 9.406944 41.618881 9.348888 41.607769 9.282776 41.59388 9.355276 41.41193 9.25083 41.366379 9.219166</gml:posList></gml:LinearRing></gml:exterior></gml:Polygon></gml:surfaceMember><gml:surfaceMember><gml:Polygon><gml:exterior><gml:LinearRing><gml:posList>50.697048 2.901285 50.773869 3.035277 50.79332 3.113055 50.741371 3.193333 50.594151 3.281388 50.538319 3.28 50.495541 3.367222 50.497211 3.602222 50.444149 3.668889 50.346371 3.671944 50.313599 3.725277 50.351929 3.763611 50.31443 4.090833 50.26527 4.21611 50.13776 4.137777 50.0811 4.226388 49.98637 4.15 49.981369 4.168332 49.938591 4.458333 49.99638 4.67361 50.129429 4.7575 50.15498 4.876388 50.07943 4.851943 49.97776 4.800832 49.9147 4.88111 49.796379 4.858054 49.768589 5.097777 49.69054 5.216944 49.698589 5.276111 49.659981 5.329721 49.63081 5.307221 49.60915 5.423332 49.5061 5.473055 49.535229 5.705208 49.527271 5.783038 49.488319 5.963333 49.448318 5.981388 49.504711 6.165277 49.45998 6.362221 49.447201 6.493888 49.363041 6.594722 49.320271 6.589722 49.162479 6.78722 49.15498 6.838888 49.21526 6.847499 49.188869 7.026666 49.11832 7.038332 49.125259 7.088332 49.14777 7.361388 49.171928 7.375277 49.16415 7.486943 49.088871 7.539999 49.04166 7.74111 49.04887 7.93861 48.968868 8.192221 48.963711 8.227394 48.88554 8.133333 48.690029 7.921515 48.59248 7.801943 48.513321 7.807499 48.49165 7.771666 48.336651 7.750833 48.156929 7.604999 48.037209 7.571666 48.002769 7.615832 47.686932 7.513333 47.58456 7.588799 47.525822 7.521388 47.469978 7.452777 47.439671 7.39561 47.44582 7.178332 47.495258 7.198054 47.497211 6.990555 47.45499 7.007777 47.444149 6.978333 47.35693 6.88111 47.371651 7.023055 47.345539 7.061665 47.30526 7.010833 47.291931 6.948332 47.068321 6.694443 47.038601 6.702777 46.987759 6.587221 46.926929 6.434721 46.774429 6.452777 46.704479 6.325022 46.593319 6.129999 46.460819 6.078055 46.40192 6.124443 46.257771 6.114444 46.250542 6.060832 46.205261 5.96611 46.17416 6 46.14027 5.965833 46.153591 6.152222 46.252491 6.305554 46.297771 6.242221 46.322392 6.243931 46.357769 6.246387 46.406929 6.328333 46.464149 6.633611 46.432491 6.79611 46.411091 6.808887 46.405499 6.804937 46.352489 6.767499 46.27137 6.844443 46.217758 6.795277 46.142208 6.788054 46.126091 6.884444 46.051929 6.871387 46.065262 6.929166 45.931721 7.038747 45.831379 6.808887 45.780819 6.799166 45.714989 6.824166 45.63998 6.990832 45.518318 6.999443 45.411652 7.181388 45.355549 7.133054 45.301929 7.122776 45.136101 6.850277 45.16832 6.760833 45.110661 6.620172 45.02026 6.746666 44.947762 6.743332 44.848591 7.009999 44.689152 7.068889 44.69915 7.000833 44.6647 6.950832 44.628868 6.960278 44.540821 6.852777 44.507771 6.857222 44.443039 6.934999 44.415821 6.88611 44.323318 6.935832 44.156651 7.279444 44.182758 7.672776 44.072208 7.704166 43.869431 7.492221 43.783451 7.534765 43.758652 7.457388 43.757824 7.4407 43.7603 7.4383 43.7631 7.4358 43.766102 7.4342 43.769199 7.4325 43.7714 7.4294 43.773102 7.4247 43.772499 7.42 43.770599 7.4161 43.768299 7.4128 43.7658 7.4097 43.763599 7.4064 43.761398 7.4031 43.758598 7.4003 43.7561 7.3972 43.753899 7.3939 43.7514 7.3906 43.7486 7.3878 43.745602 7.3864 43.741901 7.3864 43.738899 7.3883 43.735802 7.39 43.732201 7.3908 43.728031 7.391187 43.721825 7.349536 43.720539 7.345833 43.663052 7.15861 43.564159 7.133888 43.558319 7.005278 43.427212 6.854444 43.434711 6.748055 43.31554 6.669722 43.29221 6.589444 43.22443 6.681389 43.16943 6.615 43.197491 6.593888 43.149712 6.375278 43.09277 6.368055 43.12999 6.201943 43.115551 6.166666 43.056381 6.18111 43.05249 6.088611 43.07555 6.126943 43.098049 6.117499 43.133598 5.934722 43.099991 5.911111 43.057491 5.862222 43.082489 5.781111 43.139149 5.780555 43.220829 5.431944 43.221931 5.36361 43.33527 5.361666 43.36665 5.322222 43.34082 5.063611 43.3736 5.012777 43.400829 5.018332 43.464161 5.230555 43.496101 5.227221 43.47694 5.162222 43.55666 5.031388 43.496658 5.016109 43.465549 5.060833 43.430538 5.06111 43.424992 4.957777 43.42387 4.876111 43.373322 4.87361 43.41943 4.756944 43.417488 4.741666 43.37999 4.816388 43.359989 4.821666 43.361931 4.606944 43.37999 4.573333 43.424992 4.601666 43.451408 4.537837 43.455818 4.399444 43.458321 4.321111 43.509708 4.312499 43.49194 4.247222 43.471661 4.240555 43.46138 4.195276 43.498322 4.124722 43.536652 4.133055 43.56332 4.062222 43.484711 3.860555 43.278599 3.471389 43.271099 3.341111 43.084148 3.102222 43.10693 3.043333 43.069149 3.030278 43.05027 3.085278 42.940269 3.037777 42.897209 3.036944 42.84832 2.963889 42.809429 2.975277 42.798038 3.012222 42.878601 3.052499 42.553879 3.049444 42.437611 3.176962 42.433319 3.150833 42.474979 3.038333 42.401649 2.678333 42.338322 2.671111 42.35165 2.476666 42.434711 2.254167 42.363319 2.086944 42.374699 1.976666 42.437199 1.953055 42.490261 1.7875 42.473499 1.710967 42.581661 1.781667 42.65387 1.541111 42.597149 1.435247 42.689701 1.383611 42.841091 0.815833 42.839149 0.667778 42.748039 0.668055 42.689152 0.676111 42.695259 0.4125 42.675819 0.291111 42.71748 0.261944 42.694149 -0.0575 42.757488 -0.123889 42.78582 -0.187778 42.849152 -0.306667 42.796379 -0.392222 42.822208 -0.500833 42.790821 -0.519167 42.863602 -0.656111 42.965542 -0.746944 42.946091 -0.818889 43.071659 -1.300556 43.106091 -1.288333 43.031651 -1.360278 43.048321 -1.440556 43.091091 -1.472778 43.196651 -1.381945 43.273041 -1.409167 43.256378 -1.62 43.309429 -1.655833 43.385891 -1.810488 43.399429 -1.652223 43.561378 -1.493333 43.758881 -1.424445 44.259708 -1.294167 44.62582 -1.208611 44.66193 -1.192222 44.659988 -1.054445 44.683041 -1.037222 44.763599 -1.134444 44.7761 -1.168889 44.630821 -1.248889 44.691929 -1.259167 45.1236 -1.194722 45.552761 -1.095278 45.571381 -1.058333 45.520821 -1.057778 45.36721 -0.808333 45.24749 -0.735833 45.13055 -0.714722 45.0536 -0.661667 44.98666 -0.560556 45.021648 -0.589167 44.998878 -0.495 45.030819 -0.519445 45.060261 -0.62 45.124149 -0.668333 45.400269 -0.737778 45.519161 -0.85 45.70779 -1.245326 45.788052 -1.230834 45.803322 -1.146111 45.71526 -0.985278 45.839989 -1.138056 45.958881 -1.071944 46.006649 -1.111389 46.008598 -1.055556 46.100269 -1.103889 46.15527 -1.152222 46.15527 -1.2025 46.21526 -1.193889 46.258598 -1.105834 46.313599 -1.112222 46.319149 -1.195 46.273319 -1.214445 46.346661 -1.405556 46.338039 -1.448611 46.395821 -1.472222 46.419159 -1.511945 46.451931 -1.673889 46.486931 -1.799445 46.688599 -1.914167 46.813599 -2.108056 46.887211 -2.13389 47.020031 -2.011049 47.034431 -1.984445 47.09721 -2.051667 47.135818 -2.239722 47.165272 -2.163611 47.264992 -2.165556 47.29694 -2.025 47.237209 -1.815834 47.31749 -1.998889 47.30999 -2.159445 47.24165 -2.273334 47.29137 -2.495833 47.2911 -2.4425 47.322491 -2.448612 47.37693 -2.553611 47.40527 -2.396111 47.421101 -2.434167 47.48138 -2.488889 47.504162 -2.361389 47.5261 -2.662222 47.504162 -2.736111 47.49165 -2.809722 47.551659 -2.912222 47.564709 -2.877223 47.54388 -2.736945 47.613041 -2.680556 47.620541 -2.768611 47.606098 -2.88 47.59721 -2.936667 47.56443 -2.960556 47.59943 -3.125278 47.500542 -3.120278 47.521381 -3.149167 47.576649 -3.132223 47.659431 -3.200556 47.741379 -3.148056 47.652489 -3.212778 47.706661 -3.354445 47.78249 -3.281389 47.701382 -3.441389 47.811939 -3.712778 47.793598 -3.847778 47.89888 -3.951111 47.86832 -4.085556 47.881378 -4.182778 47.831928 -4.155556 47.799431 -4.209723 47.802761 -4.361945 47.8536 -4.343889 47.960819 -4.419723 48.022491 -4.529445 48.020821 -4.613056 48.040989 -4.725613 48.073601 -4.662223 48.098881 -4.295 48.178322 -4.297501 48.240551 -4.43139 48.236099 -4.493056 48.176102 -4.537223 48.233601 -4.563612 48.28194 -4.623889 48.30027 -4.513056 48.303879 -4.235001 48.319988 -4.323611 48.357769 -4.318334 48.33054 -4.448611 48.381939 -4.405556 48.427212 -4.293334 48.33527 -4.763889 48.421379 -4.790556 48.545818 -4.744167 48.579708 -4.631945 48.650539 -4.428334 48.670818 -4.314167 48.648319 -4.222778 48.6861 -4.176667 48.73193 -3.968056 48.65221 -3.953333 48.67638 -3.900278 48.629162 -3.854167 48.726101 -3.810833 48.675819 -3.596111 48.6861 -3.571111 48.72665 -3.582778 48.739159 -3.532222 48.76693 -3.580834 48.837212 -3.512778 48.8386 -3.263056 48.870541 -3.220834 48.79388 -3.224167 48.872211 -3.101667 48.826931 -3.079167 48.768879 -2.948611 48.68082 -2.883611 48.654991 -2.824722 48.572208 -2.765556 48.53054 -2.653334 48.62999 -2.464723 48.6926 -2.314336 48.669159 -2.284445 48.62471 -2.329167 48.61137 -2.226389 48.57943 -2.176389 48.646099 -2.105834 48.58305 -2.0025 48.514149 -1.9825 48.550819 -1.956944 48.650829 -2.03 48.709148 -1.849167 48.64999 -1.87 48.615551 -1.840556 48.61665 -1.638056 48.663601 -1.433889 48.744709 -1.560833 48.841099 -1.605834 48.922771 -1.549167 49.01582 -1.556667 49.008881 -1.585278 49.097759 -1.610278 49.219711 -1.549445 49.215549 -1.629445 49.381649 -1.774167 49.377769 -1.8125 49.49971 -1.845834 49.529148 -1.884444 49.57832 -1.84 49.642769 -1.851944 49.675541 -1.945833 49.723881 -1.941945 49.675541 -1.482778 49.70499 -1.410833 49.694439 -1.258333 49.586102 -1.259722 49.57888 -1.299445 49.54388 -1.301667 49.34388 -1.142222 49.394711 -1.013889 49.341099 -0.436389 49.274712 -0.218333 49.426659 0.218333 49.458881 0.393055 49.47694 0.143611 49.511662 0.078056 49.70277 0.184444 49.857491 0.611944 49.948879 1.131111 50.09832 1.435555 50.201931 1.505555 50.21944 1.555 50.184158 1.656944 50.217209 1.645 50.266651 1.548055 50.288601 1.534167 50.359989 1.552778 50.36832 1.610278 50.402489 1.558889 50.612209 1.576944 50.864429 1.576111 50.99527 1.943333 51.070541 2.472777 51.091099 2.541667 50.887211 2.6125 50.824711 2.629444 50.75555 2.781944 50.697048 2.901285</gml:posList></gml:LinearRing></gml:exterior></gml:Polygon></gml:surfaceMember></gml:MultiSurface></tows:geom>
   </tows:world>
  </gml:featureMember>
</wfs:FeatureCollection>
